OmgvaPOS/src/HelperUtils/AuthorizationHandler.cs

```diff
@@ -0,0 +1,67 @@
+﻿using OmgvaPOS.UserManagement.Enums;
+
+namespace OmgvaPOS.HelperUtils
+{
+    public static class AuthorizationHandler
+    {
+        public static bool CanManageBusiness(string? tokenString, long businessId)
```

Comment:
This could be simplified to return true if role is admin or if business id matches.

Reply:
I think it's better to be explicit who can manage a resource

Reply:
It is pretty explicit to tell that you have to be either from the same business or be an admin

Reply:
However we are not explicitly defining the roles. Basically saying "every role" if it's from the same business

Reply:
In our case "every role" works and we will probably never have other roles. However same argument could be said about id's in our entities. In our case we will never run out of int values for ids, however we chose to use longs instead